src/AnonParamsSelectionStep/AnonParamsSelectionStep.tsx

```diff
@@ -14,79 +14,42 @@ export type AnonParamsSelectionStepData = {
   anonParams: AnonymizationParams;
 };
 
-function presetAnonParams(preset: AnonParamsPreset): AnonymizationParams {
-  switch (preset) {
-    case 'P':
-      return {
-        suppression: {
-          lowThreshold: 3,
-          sD: 1.0,
-          lowMeanGap: 2.0,
-        },
-        outlierCount: { lower: 1, upper: 2 },
-        topCount: { lower: 3, upper: 4 },
-        baseNoiseSD: 1.5,
-      };
-
-    case 'XP':
-      return {
-        suppression: {
-          lowThreshold: 4,
-          sD: 1.0,
-          lowMeanGap: 2.0,
-        },
-        outlierCount: { lower: 2, upper: 3 },
-        topCount: { lower: 3, upper: 4 },
-        baseNoiseSD: 2.25,
-      };
-
-    case 'XXP':
-      return {
-        suppression: {
-          lowThreshold: 5,
-          sD: 1.0,
-          lowMeanGap: 2.0,
-        },
-        outlierCount: { lower: 2, upper: 3 },
-        topCount: { lower: 3, upper: 4 },
-        baseNoiseSD: 3.0,
-      };
-  }
-}
-
 export const AnonParamsSelectionStep: FunctionComponent<AnonParamsSelectionStepProps> = ({ children }) => {
-  const [preset, setPreset] = useState<AnonParamsPreset>('P');
-
-  const anonParams = presetAnonParams(preset);
+  const [lowThreshold, setLowThreshold] = useState<number>(3);
+  const minLowThreshold = 2;
+
+  const anonParams = {
+    suppression: {
+      lowThreshold: lowThreshold,
+      sD: 1.0,
+      lowMeanGap: 2.0,
+    },
+    outlierCount: { lower: 1, upper: 2 },
+    topCount: { lower: 3, upper: 4 },
+    baseNoiseSD: 1.5,
+  };
```

Comment:
Can you also please cleanup the anon params request structure? It serves no purpose to duplicate the default settings here if they can't vary at all.

Reply:
in similar vein to the comment on docs - this is on purpose. Earlier we've made the decision to keep the defaults which apply for DfD in here, so I think we should follow this decision.

Otherwise we'll end up with the defaults being spread across the code base. The fact that only `low_thresh` is configurable sounds like being purely up to the DfD GUI.

Reply:
The defaults are spread across the code base now. We have these values in `reference` and also here. So any change to them needs to touch both repos. I would like to avoid that.

> we've made the decision to keep the defaults which apply for DfD in here

I understood things differently: we made a decision to keep the various presets here, one of which was chosen as the default one. The option to keep them in `reference` wasn't available, as the concept of preset settings is present there.

Reply:
> The defaults are spread across the code base now. We have these values in reference and also here. So any change to them needs to touch both repos. I would like to avoid that.

I meant that all the values which apply for DfD are currently all in DfD and was referring to [this thread](https://github.com/diffix/desktop/pull/261#issuecomment-971757801)

hm, FWIW, I think it's 1/ not worth the legwork 2/ less consistent this way. I'd prefer all-or-nothing - if we set one parameter, let's set all. This way it's really easy to tweak values and configurability and iterate faster. And we can reasonably anticipate change, since we're iterating already.

But if you've got a strong opinion about this, let's change. (also mind that the current default for `low_thresh` in reference is 2 and also `top/outlierCount` are different, so we'd need to adjust there as well). Or maybe we can plant a ticket to revisit this and cleanup some time from now?

Reply:
> I meant that all the values which apply for DfD are currently all in DfD and was referring to this thread

Fair enough. But when I made that comment, all the values could vary, now most of them are fixed. To me, it makes most sense to put the default value next to the control logic (which doesn't exist anymore for most of the settings).

But ok, since you say that the `reference` defaults differ, let's make these the canonical ones.

Reply:
ok, leaving as is for now and https://github.com/diffix/desktop/issues/277 for some close future